tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -292,9 +292,11 @@ func (d *deployer) IsUp() (bool, error) {
 		d.KopsBinaryPath, "validate", "cluster",
 		"--name", d.ClusterName,
 		"--count", strconv.Itoa(d.ValidationCount),
-		"--interval", d.ValidationInterval.String(),
 		"--wait", wait.String(),
 	}
+	if d.ValidationInterval > 10*time.Second {
+		args = append(args, "--interval", d.ValidationInterval.String())
+	}
```

Comment:
For compatibility with older kOps versions that don't support this new flag

Reply:
@upodroid ^^^

Reply:
Noticed it today :(